test/apps/tangents/index.html

```diff
@@ -0,0 +1,22 @@
+<!doctype html>
```

Comment:
I personally think it is nicer to do these examples with an automatically generated HTML (html-webpack-plugin). But just follow the style of other examples in same repo.